app/helpers/materials_helper.rb

```diff
@@ -16,4 +16,10 @@ def attributes_by_material_function(material)
     material.attributes.slice(
       *DEFAULT_ATTRIBUTES+MATERIAL_ATTRIBUTES.fetch(material.function))
   end
+
+  def styled_value(value1, value2)
+    style="color: " + (value1 == value2 ? "green" : "red")
+    data = "<span style='#{style}'>#{value1}</span>"
```

Comment:
Using something like `content_tag(:span, {style: style}, value1)` would avoid the html_safe part and make this safer.  No guarantee that the material does not have html in the content.

Reply:
Done.